.gitignore

```diff
@@ -0,0 +1,41 @@
+
+# Local .terraform directories
+**/.terraform/*
+**/.terragrunt-cache
+**/.terraform.lock.hcl
```

Comment:
I'm not sure this file should be excluded from git. It ensures everybody uses exactly the same versions of providers.